modules/Story/Story.module.scss

```diff
@@ -13,4 +13,23 @@ $story-container-max-width: 720px;
     @include text-label;
 
     color: $color-base-500;
+    margin: 0;
+}
+
+.linksAndDateWrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```

Comment:
The default is left alignment, so it's better to define it in the styles. Or you could simply omit this line, which will do pretty much the same.

Reply:
Updated https://github.com/prezly/theme-nextjs-bea/pull/872/commits/d1f00798de93b1941d68af8eee97d0bd26bc54d7